src/Parser/Hunter/BeastMastery/Modules/Features/Checklist.js

```diff
@@ -0,0 +1,180 @@
+import SPELLS from 'common/SPELLS';
+import CastEfficiency from 'Parser/Core/Modules/CastEfficiency';
+import AlwaysBeCasting from 'Parser/Hunter/BeastMastery/Modules/Features/AlwaysBeCasting';
+import AMurderOfCrows from 'Parser/Hunter/BeastMastery/Modules/Talents/AMurderOfCrows';
+import SpellLink from 'common/SpellLink';
+import Icon from "common/Icon";
+import EnchantChecker from 'Parser/Core/Modules/Items/EnchantChecker';
+import DireBeast from 'Parser/Hunter/BeastMastery/Modules/Spells/DireBeast/DireBeast';
+import BestialWrathAverageFocus from 'Parser/Hunter/BeastMastery/Modules/Spells/BestialWrath/BestialWrathAverageFocus';
+
+class Checklist extends CoreChecklist {
+  static dependencies = {
+    combatants: Combatants,
+
+    //general
+    legendaryUpgradeChecker: LegendaryUpgradeChecker,
+    legendaryCountChecker: LegendaryCountChecker,
+    prePotion: PrePotion,
+
+    //features:
+    castEfficiency: CastEfficiency,
+    alwaysBeCasting: AlwaysBeCasting,
+    enchantChecker: EnchantChecker,
+
+    //talents
+    aMurderOfCrows: AMurderOfCrows,
+
+    //Bestial Wrath
+
+    //Dire Beast
+    direBeast: DireBeast,
+    bestialWrathAverageFocus: BestialWrathAverageFocus,
+
+  };
+
+  rules = [
+    new Rule({
+      name: 'Use core spells as often as possible',
+      description: <Wrapper><a href="https://www.icy-veins.com/wow/beast-mastery-hunter-pve-dps-rotation-cooldowns-abilities" target="_blank" rel="noopener noreferrer">More info.</a></Wrapper>,
+      requirements: () => {
+        const combatant = this.combatants.selected;
+        return [
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.DIRE_BEAST,
+            when: !combatant.hasTalent(SPELLS.DIRE_FRENZY_TALENT.id),
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.DIRE_FRENZY_TALENT,
+            when: combatant.hasTalent(SPELLS.DIRE_FRENZY_TALENT.id),
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.TITANS_THUNDER,
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.BESTIAL_WRATH,
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.KILL_COMMAND,
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.A_MURDER_OF_CROWS_TALENT_SHARED,
+            when: combatant.hasTalent(SPELLS.A_MURDER_OF_CROWS_TALENT_SHARED.id),
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.ASPECT_OF_THE_WILD,
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.BARRAGE_TALENT,
+            when: combatant.hasTalent(SPELLS.BARRAGE_TALENT.id),
+          }),
+        ];
+      },
+    }),
+    new Rule({
+      name: <Wrapper><SpellLink id={SPELLS.DIRE_BEAST.id} icon /> efficiency </Wrapper>,
+      description: <Wrapper>Use <SpellLink id={SPELLS.DIRE_BEAST.id} icon /> properly</Wrapper>,
```

Comment:
How do I do that? This rule *needs* more information.

Reply:
The description as it is might be better as the name of the Rule, makes it more clear what someone should do (although technically I suppose the difference is minor).